src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Controllers/Api/Public.Data/PreviewTokenControllerTests.cs

```diff
@@ -188,13 +256,17 @@ await TestApp.AddTestData<PublicDataDbContext>(context =>
         private async Task<HttpResponseMessage> CreatePreviewToken(
             Guid dataSetVersionId,
             string label,
-            HttpClient? client = null)
+            HttpClient? client = null,
+            DateTimeOffset? activates = null,
+            DateTimeOffset? expires = null)
```

Comment:
You've added tests that use these in an invalid way and check for an error, but there's no tests which use them in a valid way.

I think ideally there would be tests that check a token is created with a 

- valid `activates` and no `expiry`
- valid `activates` and valid `expiry`
- no `activates` and valid `expiry`

`Success` already checks a token can be created with no `activated` and no `expiry`.